### PR TITLE
Include parameter to change number of sensitive attributes

### PR DESCRIPTION
**Pull request by asim29**
The code had a hard-coded value for the number of sensitive attributes. CelebA only had one sensitive attribute (based on @vasishtduddu's code). Needed to generalize it to make it work.

### CODE REVIEW

File: amulet/discriminatory_behavior/defenses/adversarial_debiasing.py [hunk 1]
```diff
@@ -47,6 +57,10 @@ class AdversarialDebiasing(DicriminatoryBehaviorDefense):
             Training data loader to adversarial training.
         test_loader: :class:`~torch.utils.data.DataLoader`
             Testing data loader to adversarial training.
+        n_sensitive: int
+            Number of sensitive attributes in the dataset.
+        n_input: int
```
Comment (sebszyller) [inline amulet/discriminatory_behavior/defenses/adversarial_debiasing.py:62]: `__init__` says `n_classes`

File: amulet/discriminatory_behavior/defenses/adversarial_debiasing.py [hunk 2]
```diff
@@ -62,6 +76,8 @@ def __init__(
         optimizer: torch.optim.Optimizer,
         train_loader: DataLoader,
         test_loader: DataLoader,
+        n_sensitive: int,
+        n_classes: int,
```
[the remark this answers is not in the document]
Comment (asim29) [reply amulet/discriminatory_behavior/defenses/adversarial_debiasing.py:80]: Do we? Theoretically, can't we predict any number of sensitive attributes regardless of the number of classes?

[the remark this answers is not in the document]
Comment (sebszyller) [reply amulet/discriminatory_behavior/defenses/adversarial_debiasing.py:80]: you're right, I misread; we'd rename it `n_sensitive_attributes` or some abbreviated version of it though.

[the remark this answers is not in the document]
Comment (asim29) [reply amulet/discriminatory_behavior/defenses/adversarial_debiasing.py:80]: done

**Comment by sebszyller** [comment]
merge away

**Pull request merged**